openapi/nexus.json

```diff
@@ -10499,120 +10611,574 @@
+            "required": [
+              "multiple_labels"
+            ],
+            "additionalProperties": false
           },
-          "name": {
-            "$ref": "#/components/schemas/Name"
+          {
+            "description": "RFC 9234 TODO",
+            "type": "object",
+            "properties": {
+              "bgp_role": {
+                "type": "object"
+              }
+            },
+            "required": [
+              "bgp_role"
+            ],
+            "additionalProperties": false
           },
-          "service": {
-            "description": "The service using this certificate",
-            "allOf": [
-              {
-                "$ref": "#/components/schemas/ServiceUsingCertificate"
+          {
+            "description": "RFC 4724 TODO",
+            "type": "object",
+            "properties": {
+              "graceful_restart": {
+                "type": "object"
               }
-            ]
-          }
-        },
-        "required": [
-          "cert",
-          "description",
-          "key",
-          "name",
-          "service"
-        ]
-      },
-      "CertificateResultsPage": {
-        "description": "A single page of results",
-        "type": "object",
-        "properties": {
-          "items": {
-            "description": "list of items on this page of results",
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/Certificate"
-            }
+            },
+            "required": [
+              "graceful_restart"
+            ],
+            "additionalProperties": false
           },
-          "next_page": {
-            "nullable": true,
-            "description": "token used to fetch the next page of results (if any)",
-            "type": "string"
-          }
-        },
-        "required": [
-          "items"
-        ]
-      },
-      "Cumulativedouble": {
-        "description": "A cumulative or counter data type.",
-        "type": "object",
-        "properties": {
-          "start_time": {
-            "type": "string",
-            "format": "date-time"
+          {
+            "description": "RFC 6793",
+            "type": "object",
+            "properties": {
+              "four_octet_as": {
+                "type": "object",
+                "properties": {
+                  "asn": {
+                    "type": "integer",
+                    "format": "uint32",
+                    "minimum": 0
+                  }
+                },
+                "required": [
+                  "asn"
+                ]
+              }
+            },
+            "required": [
+              "four_octet_as"
+            ],
+            "additionalProperties": false
           },
-          "value": {
-            "type": "number",
-            "format": "double"
-          }
-        },
-        "required": [
+          {
+            "description": "draft-ietf-idr-dynamic-cap TODO",
```

Comment:
are these TODOs things we want to address before shipping? It would be visible in the docs